internal/plugins/ansible/v1/plugin.go

```diff
@@ -16,17 +16,17 @@ package ansible
 
 import (
 	"sigs.k8s.io/kubebuilder/v3/pkg/config"
+	cfgv3 "sigs.k8s.io/kubebuilder/v3/pkg/config/v3"
 	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
 
 	"github.com/operator-framework/operator-sdk/internal/plugins"
-	cfgv3 "sigs.k8s.io/kubebuilder/v3/pkg/config/v3"
 )
 
-const pluginName = "ansible" + plugins.DefaultNameQualifier
+const pluginName = "base.ansible" + plugins.DefaultNameQualifier
```

Comment:
imo: nit we can add a comment explaining why we're using the base and etc .. and that is not reflected in the end result on the PROJECT file.